src/rgw/rgw_rados.cc

```diff
@@ -4000,7 +4000,14 @@ int RGWRados::open_gc_pool_ctx()
 
 int RGWRados::open_lc_pool_ctx()
 {
-  const char *lc_pool = get_zone_params().lc_pool.name.c_str();
+  const char *lc_pool;
+
+  if (!get_zone_params().lc_pool.name.empty()){
+    lc_pool = get_zone_params().lc_pool.name.c_str();
+  } else {
+    lc_pool = "default.rgw.lc";
```

Comment:
i'm worried about the "default" part here. in a multisite configuration, we can't have multiple zones using the same "default.rgw.lc" pool. we avoid this by having each zone's pool names start with the zone name instead of "default"

so `lc_pool` should default to `get_zone_params().get_name() + ".rgw.lc"`. and i think we're better off setting this default in `RGWZoneParams::decode()` when we find that `struct_v < 7`